lib/workers/repository/init/index.ts

```diff
@@ -21,6 +21,7 @@ export async function initRepo(
   config = await getRepoConfig(config);
   checkIfConfigured(config);
   await setBranchPrefix(config.branchPrefix);
+  setIgnoredAuthors(config.gitIgnoredAuthors);
```

Comment:
@rarkins Maybe we want to join those to a single configure method?

Reply:
Maybe setRepoUserConfig()?